src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Validation/ValidatorCache.cs

```diff
@@ -4,16 +4,29 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Diagnostics;
-using Microsoft.AspNetCore.Mvc.ModelBinding;
-using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+using Microsoft.AspNetCore.Mvc.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.Internal
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Validation
 {
+    /// <summary>
+    /// Provides and caches <see cref="IModelValidator"/>s for <see cref="ModelMetadata"/>.
+    /// </summary>
     public class ValidatorCache
     {
-        private readonly ConcurrentDictionary<ModelMetadata, CacheEntry> _cacheEntries = new ConcurrentDictionary<ModelMetadata, CacheEntry>();
-
-        public IReadOnlyList<IModelValidator> GetValidators(ModelMetadata metadata, IModelValidatorProvider validatorProvider)
+        private readonly ConcurrentDictionary<ModelMetadata, CacheEntry> _cacheEntries
+            = new ConcurrentDictionary<ModelMetadata, CacheEntry>();
+
+        /// <summary>
+        /// Provides <see cref="IModelValidator"/>s for <paramref name="metadata"/>.
+        /// </summary>
+        /// <param name="metadata">The <see cref="ModelMetadata"/>.</param>
+        /// <param name="validatorProvider">
+        /// The <see cref="IModelValidatorProvider"/> to use in case of a cache miss.
+        /// </param>
+        /// <returns>List of <see cref="IModelValidator"/>s for <paramref name="metadata"/>.</returns>
+        public IReadOnlyList<IModelValidator> GetValidators(
+            ModelMetadata metadata,
+            IModelValidatorProvider validatorProvider)
```

Comment:
Nope. But, this isn't that relevant because the PR no longer makes `ValidatorCache` public.